packages/frontend/src/MobileRoutes.tsx

```diff
@@ -0,0 +1,264 @@
+import posthog from 'posthog-js';
+import React, { FC, useCallback, useState } from 'react';
+import { Link, Redirect, Route, Switch } from 'react-router-dom';
+import AppRoute from './components/AppRoute';
+import MantineIcon from './components/common/MantineIcon';
+import RouterNavLink from './components/common/RouterNavLink';
+import ForbiddenPanel from './components/ForbiddenPanel';
+import MobileView from './components/Mobile';
+import ProjectSwitcher from './components/NavBar/ProjectSwitcher';
+import PrivateRoute from './components/PrivateRoute';
+import ProjectRoute from './components/ProjectRoute';
+import { useActiveProjectUuid } from './hooks/useActiveProject';
+import useLogoutMutation from './hooks/user/useUserLogoutMutation';
+import Login from './pages/Login';
+import MinimalDashboard from './pages/MinimalDashboard';
+import MinimalSavedExplorer from './pages/MinimalSavedExplorer';
+import MobileCharts from './pages/MobileCharts';
+import MobileDashboards from './pages/MobileDashboards';
+import MobileHome from './pages/MobileHome';
+import MobileSpace from './pages/MobileSpace';
+import MobileSpaces from './pages/MobileSpaces';
+import Projects from './pages/Projects';
+import ShareRedirect from './pages/ShareRedirect';
+import { TrackPage } from './providers/TrackingProvider';
+import { ReactComponent as Logo } from './svgs/logo-icon.svg';
+import { PageName } from './types/Events';
+
+const MobileNavBar: FC = () => {
+    const [isMenuOpen, setIsMenuOpen] = useState(false);
+    const toggleMenu = useCallback(
+        () => setIsMenuOpen((prevValue) => !prevValue),
+        [],
+    );
+    const { activeProjectUuid } = useActiveProjectUuid({
+        refetchOnMount: true,
+    });
+    const { mutate: logout } = useLogoutMutation({
+        onSuccess: () => {
+            posthog.reset();
+            window.location.href = '/login';
+        },
+    });
+
+    return (
+        <MantineProvider inherit theme={{ colorScheme: 'dark' }}>
+            <Header
+                height={50}
+                display="flex"
+                px="md"
+                zIndex={getDefaultZIndex('app')}
+                sx={{
+                    alignItems: 'center',
+                    boxShadow: 'lg',
+                }}
+            >
+                <Group align="center" position="apart" sx={{ flex: 1 }}>
+                    <ActionIcon
+                        component={Link}
+                        to={'/'}
+                        title="Home"
+                        size="lg"
+                    >
+                        <Logo />
+                    </ActionIcon>
+                    <Burger opened={isMenuOpen} onClick={toggleMenu} />
+                </Group>
+            </Header>
+            <Drawer opened={isMenuOpen} onClose={toggleMenu} size="75%">
+                <Title order={6} fw={600} mb="xs">
+                    Project
+                </Title>
+                <ProjectSwitcher />
```

Comment:
not a blocker, just a note: 
When I switch `projects` the switcher disappears, and appears again

Reply:
Not fixing it in this PR.